extensions/roam-research/package.json

```diff
@@ -2,59 +2,122 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "roam-research",
   "title": "Roam Research",
-  "description": "Roam Research With Raycast",
-  "icon": "favicon.png",
-  "author": "dive2Pro",
+  "description": "Your Roam graphs at your fingertips! Quickly search, capture and navigate your Roam graphs.",
+  "icon": "icon.png",
+  "author": "roamresearch",
```

Comment:
transferring ownership of the extension to "roamresearch" user ( I work at Roam and I created that Raycast user). I've added @dive2Pro to the contributors list and also have gotten their blessing for this transfer. Can you confirm, @dive2Pro ?

Reply:
> transferring ownership of the extension to "roamresearch" user ( I work at Roam and I created that Raycast user). I've added @dive2Pro to the contributors list and also have gotten their blessing for this transfer. Can you confirm, @dive2Pro ?

Yes, I confirm. I'd love to transfer this repo to "roamresearch